recipes/sentry-crashpad/all/conandata.yml

```diff
@@ -1,67 +1,17 @@
 sources:
+  "0.4.17":
+    url: "https://github.com/getsentry/sentry-native/releases/download/0.4.17/sentry-native.zip"
+    sha256: "38c9e180c29b6561c5f91ada154e191eabe9d7c270734cff81e55532878ec273"
   "0.4.15":
     url: "https://github.com/getsentry/sentry-native/releases/download/0.4.15/sentry-native.zip"
     sha256: "ae3ac4efa76d431d8734d7b0b1bf9bbedaf2cbdb18dfc5c95e2411a67808cf29"
-  "0.4.14":
```

Comment:
Why did you remove 14 but kept 13?

Reply:
Because this recipe is a dependency of sentry-native which doesn't have 0.4.14 version: https://github.com/conan-io/conan-center-index/blob/master/recipes/sentry-native/config.yml

Reply:
Good catch!